test/scripts/install-kind.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -ex
+
+export GO111MODULE="on"
+mkdir -p $GOPATH/bin
+curl -fs https://chunk.io/trozet/ba750701d0af4e2b94b249ab9de27b50 -o $GOPATH/bin/kubetest
+chmod +x $GOPATH/bin/kubetest
+
+pushd $GOPATH/src/k8s.io/kubernetes/
+sudo ln ./_output/local/go/bin/kubectl /usr/local/bin/kubectl
+popd
+
+GO111MODULE="on" go get sigs.k8s.io/kind@v0.7.0
```

Comment:
Instead of building this with go get, maybe we could be a little more efficient here and just download the binary directly?
curl -Lo ./kind "https://github.com/kubernetes-sigs/kind/releases/download/v0.7.0/kind-$(uname)-amd64"
chmod +x ./kind
mv ./kind /some-dir-in-your-PATH/kind

https://github.com/kubernetes-sigs/kind#installation-and-usage

Reply:
Opened #1215 as a reminder to address this.